--- src/textedit/object.h	2014-11-09 12:25:35.758135819 +0800
+++ src/textedit/object.h	2014-11-09 12:26:00.035033742 +0800
@@ -272,7 +272,7 @@
 		va_list va;
 		DWORD   dva;
 	}_va;
-	_va.va = va;
+	va_copy(_va.va , va);	
 	if(_va.dva == 0)
 		return 0;
 	//return va_arg(va, int); 
--- src/newgal/pcxvfb/Makefile	2014-11-09 21:18:59.196083504 +0800
+++ src/newgal/pcxvfb/Makefile	2014-11-09 21:21:31.635256877 +0800
@@ -192,7 +192,7 @@
 top_builddir = ../../..
 top_srcdir = ../../..
 INCLUDES = -I$(abs_top_srcdir)/src/include -I$(abs_top_srcdir)/include \
-			            -I$(abs_top_srcdir)/src/newgal/ -I/usr/include/
+			            -I$(abs_top_srcdir)/src/newgal/
 
 noinst_LTLIBRARIES = libvideo_pcxvfb.la
 libvideo_pcxvfb_la_SOURCES = $(PCXVFB_SRCS)
